client/src/components/Feed/NestedComments.js

```diff
@@ -51,14 +58,12 @@ const NestedComments = ({ comment, dispatchPostAction, deleteComment }) => {
   const actorId = useSelector(selectActorId);
 
   const renderAvatar = (
-    <Avatar
-      src={
-        comment.author.photo
-          ? comment.author.photo
-          : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTGhWTUkY0xGbbdHyReD6227iz53ADtRmcn1PTN4GUS3clC6MCT&usqp=CAU"
-      }
-      alt={`${comment.author.name}`}
-    />
+    <Avatar src={comment.author.photo} alt={`${comment.author.name}`}>
+      {!comment.author.photo &&
```

Comment:
A tip for the futur: This check is not necessary since Avatar from "antd" already have some similar logic in place.
This is from their docs:
> Tip: You can set `icon` or `children` as the fallback for image load error, with the priority of `icon` > `children`

Reply:
Got it @Zaydme , thanks for the advice 👍